### PR TITLE
add possibility to disable language switch

### PR DESCRIPTION
**Pull request by fm-sys**
grafik

Unchecking the checkbox will disable the language switch. (default=enabled)

### CODE REVIEW

File: .idea/checkstyle-idea.xml [hunk 1]
```diff
@@ -16,4 +16,4 @@
       </list>
     </option>
   </component>
-</project>
+</project>
```
Comment (moving-bits) [inline .idea/checkstyle-idea.xml:19]: Is this change included deliberately?

Comment (fm-sys) [reply .idea/checkstyle-idea.xml]: Not really. Android studio had changed the file automatically. I just reverted it...

**Comment by fm-sys** [comment]
I found some replacements for language dependent parsing and included them into the PR as well...

**Comment by fm-sys** [comment]
retest this please

**Comment by fm-sys** [comment]
The error seems kinda unrelated:
```
FAILURE: Build failed with an exception.
00:16:38  
00:16:38  * What went wrong:
00:16:38  Execution failed for task ':main:lintBasicDebug'.
00:16:38  > Lint found errors in the project; aborting build.
00:16:38    
00:16:38    Fix the issues identified by lint, or add the following to your build script to proceed with errors:
00:16:38    ...
00:16:38  > Task :main:lintBasicDebug FAILED
00:16:38    android {
00:16:38        lintOptions {
00:16:38            abortOnError false
00:16:38        }
00:16:38    }
00:16:38    ...
00:16:38    Errors found:
00:16:38    
00:16:38    res/values-lt/strings.xml:1404: Error: Inconsistent formatting types for argument #4 in format string init_renderthemefolder_synctolocal_dialog_message ('% d'): Found both 's' and 'd' (in values/strings.xml) [StringFormatMatches]
00:16:38        <string name="init_renderthemefolder_synctolocal_dialog_message">Taip bus nuolat sinchronizuojamas žemėlapio temos aplanko turinys į privatų programos aplanką, kad failai būtų greičiau pasiekti. Prieš įjungdami šią funkciją įsitikinkite, kad temos aplanke yra tik jūsų temos, bet nėra papildomų (didelių) duomenų.\n\nJūsų žemėlapio temos aplanke \'%1$s\' šiuo metu yra %2$s, %3$s ir % dydis %4$s.\n\nAr norite įjungti aplankų sinchronizavimą?</string>
00:16:38                                                                                                                                                                                                                                                                                                                                                                                                       ~~~
00:16:38        res/values/strings.xml:1707: Conflicting argument type here
```

**Comment by fm-sys** [comment]
retest this please

**Comment by fm-sys** [comment]
Note: this setting should be hidden/removed before releasing a new version from `master`.

**Pull request merged**